src/components/explore-section/ExploreInteractive/DataTypeGroupTotals.tsx

```diff
@@ -87,10 +100,36 @@ export default function DataTypeGroupTotals({
           href={`${pathName}/model/circuit`}
           key="Circuit"
           title="Circuit"
-          subtitle={`${circuitsFlat.length} records`}
+          subtitle={`${circuitMap.size} record${circuitMap.size !== 1 ? 's' : ''}`}
           testId="experiment-dataset-Circuit"
         />
       )}
     </>
   );
 }
+
+export const getStaticProps: GetStaticProps<DataTypeGroupTotalsProps> = async () => {
```

Comment:
This seems to be breaking compilation: the file is marked as a client component but contains a global import for `fs/promises` module that is only available in the Nodejs environment.

I presume we need to use fetch in the client components and can read fs directly only in server components.

Reply:
The page that is failing right now: http://localhost:3000/app/virtual-lab/explore/interactive